dev-java/cortado/cortado-0.6.0-r2.ebuild

```diff
@@ -0,0 +1,60 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit java-pkg-2 java-ant-2
+
+DESCRIPTION="Multimedia framework for Java written by Fluendo"
+HOMEPAGE="https://www.theora.org/cortado/"
+SRC_URI="https://downloads.xiph.org/releases/cortado/${P}.tar.gz"
+
+LICENSE="GPL-2 LGPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc64 ~x86"
+
+# To prevent following errors we restrict to jdk:1.8
+
+# The ' characters around the executable and arguments are
+# not part of the command.
+#     [javac] /var/tmp/portage/dev-java/cortado-0.6.0-r2/work/cortado-0.6.0/src/com/fluendo/plugin/AudioSinkSA.java:22: error: package sun.audio does not exist
+#     [javac] import sun.audio.*;
+#     [javac] ^
+#     [javac] /var/tmp/portage/dev-java/cortado-0.6.0-r2/work/cortado-0.6.0/src/com/fluendo/plugin/AudioSinkSA.java:93: error: cannot find symbol
+#     [javac]     private AudioStream stream;
+#     [javac]             ^
+#     [javac]   symbol:   class AudioStream
+#     [javac]   location: class AudioSinkSA.RingReader
+#     [javac] /var/tmp/portage/dev-java/cortado-0.6.0-r2/work/cortado-0.6.0/src/com/fluendo/plugin/AudioSinkSA.java:100: error: cannot find symbol
+#     [javac]         stream = new AudioStream(this);
+#     [javac]                      ^
+#     [javac]   symbol:   class AudioStream
+#     [javac]   location: class AudioSinkSA.RingReader
+#     [javac] /var/tmp/portage/dev-java/cortado-0.6.0-r2/work/cortado-0.6.0/src/com/fluendo/plugin/AudioSinkSA.java:107: error: package AudioPlayer does not exist
+#     [javac]       AudioPlayer.player.start(stream);
+#     [javac]                  ^
+#     [javac] /var/tmp/portage/dev-java/cortado-0.6.0-r2/work/cortado-0.6.0/src/com/fluendo/plugin/AudioSinkSA.java:111: error: package AudioPlayer does not exist
+#     [javac]       AudioPlayer.player.stop(stream);
+#     [javac]                  ^
+#     [javac] /var/tmp/portage/dev-java/cortado-0.6.0-r2/work/cortado-0.6.0/src/com/fluendo/plugin/AudioSinkSA.java:115: error: package AudioPlayer does not exist
+#     [javac]       AudioPlayer.player.stop(stream);
+#     [javac]                  ^
+DEPEND="virtual/jdk:1.8"
+RDEPEND=">=virtual/jre-1.8:*"
+
+EANT_BUILD_TARGET="stripped"
+
+src_prepare() {
+	default
+	echo "#!/bin/sh" > scripts/get-revision || die
```

Comment:
This should use a "here document", instead of invoking `echo` twice. See https://devmanual.gentoo.org/tools-reference/cat/#here-documents

Reply:
Done